cmd/kubeadm/app/constants/constants.go

```diff
@@ -171,6 +171,15 @@ const (
 
 	// DefaultCIImageRepository points to image registry where CI uploads images from ci-cross build job
 	DefaultCIImageRepository = "gcr.io/kubernetes-ci-images"
+
+	// KubeControllerManagerOptionsAddress represents the address argument of the kube-controller-manager configuration.
```

Comment:
Oh yeah, sorry if you mis-interpreted my comment earlier, could you move these as constants in the same `utils/staticpod/utils.go` file?